dashboard/app/controllers/activities_controller.rb

```diff
@@ -208,15 +208,15 @@ def trophy_check(user)
       pct = counts[:current].to_f/counts[:max]
 
       new_trophy = Trophy.find_by_id case
-        when pct == Trophy::GOLD_THRESHOLD
-          Trophy::GOLD
-        when pct >= Trophy::SILVER_THRESHOLD
-          Trophy::SILVER
-        when pct >= Trophy::BRONZE_THRESHOLD
-          Trophy::BRONZE
-        else
-          # "no trophy earned"
-      end
+                                     when pct == Trophy::GOLD_THRESHOLD
+                                       Trophy::GOLD
+                                     when pct >= Trophy::SILVER_THRESHOLD
+                                       Trophy::SILVER
+                                     when pct >= Trophy::BRONZE_THRESHOLD
+                                       Trophy::BRONZE
+                                     else
+                                       # "no trophy earned"
+                                     end
```

Comment:
Doing so increases the number of existing violations, so unless there is a stylistic reason to switch, I'm going to vote we keep the default.